src/vllm_router/dynamic_config.py

```diff
@@ -145,7 +163,11 @@ def reconfigure_service_discovery(self, config: DynamicRouterConfig):
                 app=self.app,
                 urls=parse_static_urls(config.static_backends),
                 models=parse_comma_separated_args(config.static_models),
-                aliases=parse_comma_separated_args(config.static_aliases),
+                aliases=(
+                    parse_static_aliases(config.static_aliases)
+                    if config.static_aliases
+                    else None
+                ),
```

Comment:
We forgot to add this in https://github.com/vllm-project/production-stack/pull/426

Reply:
Isn't that handled inside `parse_comma_separated_args`

Reply:
@max-wittig Nope, the logic is not the same. Also, we should return a `dict` and not a `list`:

```python
def parse_comma_separated_args(comma_separated_string: Optional[str]):
    if comma_separated_string is None:
        return None
    return comma_separated_string.split(",")

def parse_static_aliases(static_aliases: str):
    aliases = {}
    for alias_and_model in static_aliases.split(","):
        alias, model = alias_and_model.split(":")
        aliases[alias] = model
    logger.info(f"Loaded aliases {aliases}")
    return aliases
```